packages/opencv/lib/index.js

```diff
@@ -682,3 +587,130 @@ export {getImagesMatches, getImagesSimilarity, getImageOccurrence, initOpenCv};
  * @property {any} KeyPointVector
  * @property {any} FeatureDetector
  */
+
+/**
+ * @typedef SimilarityOptions
+ * @property {boolean} [visualize=false] Whether to return the resulting visalization
+ * as an image (useful for debugging purposes)
+ * @property {string} [method=TM_CCOEFF_NORMED] The name of the template matching method.
+ * Acceptable values are:
+ * - `TM_CCOEFF`
+ * - `TM_CCOEFF_NORMED` (default)
+ * - `TM_CCORR`
+ * - `TM_CCORR_NORMED`
+ * - `TM_SQDIFF`
+ * - `TM_SQDIFF_NORMED`
+ * Read https://docs.opencv.org/3.0-beta/doc/py_tutorials/py_imgproc/py_template_matching/py_template_matching.html
+ * for more details.
+ */
+
+/**
+ * @typedef SimilarityResult
+ * @property {number} score The similarity score as a float number in range [0.0, 1.0].
+ * 1.0 is the highest score (means both images are totally equal).
+ * @property {Buffer} [visualization] The visualization of the matching result
```

Comment:
it won't be buffer anymore after we call `toString()` on it

Reply:
ah, you're right.  there's no strict type checking going on here.